From a36476f6a9d376afd6eceb34b8a8f443bc37cdb1 Mon Sep 17 00:00:00 2001
From: Potin Lai <potin.lai.pt@gmail.com>
Date: Fri, 26 Jul 2024 19:03:48 +0800
Subject: [PATCH 04/14] ARM: dts: aspeed: catalina: add extra ncsi properties

Add extra properties which only avaliable in Meta common patches.

- ncsi-ctrl,start-redo-probe
  - patch: common/recipes-kernel/linux/6.6/0173-NCSI-Add-propety-no-channel-monitor-and-start-redo-p.patch
- ncsi-package
  - patch: common/recipes-kernel/linux/6.6/0174-net-ncsi-specify-maximum-package-to-prob.patch

Signed-off-by: Potin Lai <potin.lai.pt@gmail.com>
---
 arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts
index 8d786510167f5..d007e858f4c31 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts
@@ -158,6 +158,8 @@ &mac2 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_ncsi3_default>;
 	use-ncsi;
+	ncsi-ctrl,start-redo-probe;
+	ncsi-package = <1>;
 };
 
 &mac3 {
@@ -165,6 +167,8 @@ &mac3 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_ncsi4_default>;
 	use-ncsi;
+	ncsi-ctrl,start-redo-probe;
+	ncsi-package = <2>;
 };
 
 &fmc {
-- 
2.31.1

